src/main/java/com/ensao/gi5/lint/wrapper/CompilationUnitWrapper.java

```diff
@@ -1,27 +1,41 @@
 package com.ensao.gi5.lint.wrapper;
 
+import com.github.javaparser.JavaParser;
 import com.github.javaparser.Problem;
 import com.github.javaparser.ast.CompilationUnit;
 import com.github.javaparser.ast.ImportDeclaration;
+import com.github.javaparser.ast.Node;
 import com.github.javaparser.ast.NodeList;
+import com.github.javaparser.ast.expr.VariableDeclarationExpr;
 import com.github.javaparser.ast.visitor.VoidVisitor;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
+import java.io.File;
+import java.io.IOException;
+import java.util.*;
 
 public class CompilationUnitWrapper {
-	private final CompilationUnit compilationUnit;
-	private final String fileName;
+	private  CompilationUnit compilationUnit;
+	private  String fileName;
 	private final List<Problem> problems = new ArrayList<>();
 
 	public CompilationUnitWrapper(CompilationUnit compilationUnit, String fileName) {
 		this.compilationUnit = compilationUnit;
 		this.fileName = fileName;
 	}
-	
-	public NodeList<ImportDeclaration> getImports() {return compilationUnit.getImports();}
 
+	public CompilationUnitWrapper(File file) {
+		try {
+			this.compilationUnit = new JavaParser().parse(file).getResult().get();
+			this.fileName = file.getName();
+		} catch (IOException e) {
+			e.printStackTrace();
```

Comment:
`e.printStackTrace()` is for debug purposes, provide a context for this exception, like adding message, or propagating it to the next level.
``` java
log.info("Error while reading file " file.getAbsolutePath(), e);
```